.github/workflows/vvv-provisioning.yml

```diff
@@ -30,13 +30,18 @@ jobs:
           sudo apt-get -q --autoremove --purge remove php*
           sudo apt-get -q autoclean
 
+          # remove pre-installed composer
+          if [ -f /usr/bin/composer ]; then
+            sudo rm -f /usr/bin/composer
```

Comment:
This doesn't fully install the previously installed composer. Just removes it's executable. Shouldn't we also remove it's config and such?

Reply:
No. I discussed this in a bit more on [the 20.04 issue](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/2389#issuecomment-815518165), but this only runs in the GH action. We _have_ to remove the bin in some environments (20.x+ actions include that file, which breaks our installer), but our composer doesn't have issues with any other files from the base install (honestly, I don't think there really are any), so the less we have to run the faster actions will complete.

Reply:
If it is just for the GH action I don't think that is a problem.